src/components/notes/List/NoteRow.jsx

```diff
@@ -137,16 +136,13 @@ const NoteRow = ({ note, f, t, client }) => {
           <Link
             className={styles.actionMenuItem}
             onClick={closeMenu}
-            to={{
-              pathname: `/${Routes.ShareFromList}`,
-              state: {
-                background: location,
-                modalProps: {
-                  document: { ...note, name: note.attributes.name },
-                  documentType: DocumentTypes.Files,
-                  onClose: history.goBack,
-                  sharingDesc: note.attributes.name
-                }
+            to={AppRoutes.ShareFromList}
+            state={{
+              backgroundLocation: location,
+              modalProps: {
```

Comment:
maybe rename to `shareModal`

Reply:
Also maybe it would be better if the share modal could fetch itself all these params from the location params (noteId), that would remove the need to juggle stateful data, we shouldn't be doing that